src/components/vulnerabilities/Form.js

```diff
@@ -21,6 +24,7 @@ const VulnerabilityForm = ({
     const [projects, setProjects] = useState(null);
     const [categories, setCategories] = useState(null);
     const [targets, setTargets] = useState(null);
+    const [useOWASP, setMetrics] = useState(false);
```

Comment:
I think this could be clearer defined in this way:
```js
const [riskRatingMethodology, setRiskRatingMethodology] = useState(ProjectVulnerabilityMetrics[0].id);
```
Or something along those lines. This is a minor thing and as such not a blocker for merging it 👍🏼 . I might rename it after. Thanks!

Reply:
Yeah, as I am reading it it definitely makes sense. My name of the value is not chosen very well. What you are proposing sound more appropriate. Thank you!